gravitee-apim-portal-webui/src/app/app.component.html

```diff
@@ -101,7 +101,7 @@ <h1>{{ this.homepageTitle }}</h1>
         <li>
           <gv-link path="cookies" title="{{ 'route.cookies' | translate }}"></gv-link>
         </li>
-        <li>{{ 'footer.rights' | translate }}</li>
+        <li>{{ 'footer.rights' | translate }} {{ currentYear }}</li>
```

Comment:
question: I don't know if it's worth setting a range of years?
```suggestion
        <li>{{ 'footer.rights' | translate }} 2020-{{ currentYear }}</li>
```

Reply:
No idea what's the best practice for this

Reply:
Looks like there is no strong requirement on this so I'll merge as it is